### DISCUSSION TITLE
Friendlier Click-Drag Scrolling

### DISCUSSION THREAD

**Discussion by akett** [vonbeardy/hd2galaxy-issues#26] [Ideas]
![Screenshot 2024-03-29 214931](https://github.com/vonbeardy/hd2galaxy-issues/assets/6234564/7e3e4a56-7490-4e7f-b58e-edbc1cad0539)

Click-dragging the map and mousing over your portal/affix elements causes the dragging to stop and highlights the text, the result of which is pictured above. This is a minor issue to be sure, but it should be a simple fix if you'd like to change it!

When dragging is active, just add pointer-events: none; to your .mantine-Affix-root class, then remove it when dragging stops or the mouse goes off screen, the window loses focus, etc.

You'll know best for how to implement it in your app, I'm just throwing out a simple solution for you to mull over if you decide it's even an issue. Thanks for the app btw!

**Comment by vonbeardy** [comment]
This is kind annoying, not sure I had ran across this. I'll make an issue related to this. I did notice with the tooltips on the planets when highlighting text also continue to drag the map when it shouldn't.
